### ISSUE TITLE
Create helm tests for worker/server charts

### ISSUE THREAD

**Issue by jamiezieziula** [PrefectHQ/prefect-helm#304]
as defined [here](https://helm.sh/docs/topics/chart_tests/)

**Comment by parkedwards** [comment]
oh @jamiezieziula - we can `kubectl logs` by deployment name, eg.

```
kubectl logs deployment/shoveler | grep something-interesting
```

**Comment by ialejandro** [comment]
@jamiezieziula can help with this example? [helm lint-test](https://github.com/devops-ia/helm-charts/blob/main/.github/workflows/lint-test.yaml), the main config [ct.yaml](https://github.com/devops-ia/helm-charts/blob/main/ct.yaml) put on root dir, and you can define CI values per chart (example: [ci-values.yaml](https://github.com/devops-ia/helm-charts/blob/main/charts/opencti/ci/ci-values.yaml))

More info.: https://github.com/helm/chart-testing

**Comment by mitchnielsen** [comment]
Thanks for the pointer @ialejandro 🤝 We have since implemented `chart-testing` to some degree (see https://github.com/PrefectHQ/prefect-helm/blob/db-secret-config/.github/linters/server-ct.yaml, for example). Looks like it's been helpful for validating that it can install the chart properly.

I see it can lint as well, but on that note: as part of https://github.com/PrefectHQ/prefect-helm/pull/365, I tried [helm-unittest](https://github.com/helm-unittest/helm-unittest) and have had a great experience with it so far. It's pretty easy to make a set of tests and confirm the output is what we expect. This can also serve as example configuration values for end users with varying use cases.

If the feedback in that PR is positive, then once it's merged I'll follow up and further implement tests for the worker and server charts.
